2주차/20920.py

```diff
@@ -0,0 +1,12 @@
+N,M = map(int,input().split())
+word =[input() for _ in range(N)]
+for w in word :
+    if len(w) < M :
+        word.remove(w)
+count={x:word.count(x) for x in word}
```

Comment:
```suggestion
counts={}
for x in word:
    if x in counts:
        counts[x] += 1
    else:
        counts[x] = 1
```
이부분도 word.count(x)의 시간복잡도가 O(n)으로 딕셔너리 컴프리헨션 전체의 시간 복잡도는 O(n^2)이 돼요.
갯수 카운트하는 딕셔너리를 만들때는 스터디때 소개해준 Counter를 사용하거나 수정한 코드처럼 입력받아보세요.

그리고 count라는 딕셔너리를 만들어버리면 count 함수가 덮어쓰기 돼버리니까 웬만하면 내장함수 이름이랑 변수명은 겹치지 않게 해주세요